clonelists/Sony - PlayStation (Redump).json

```diff
@@ -3254,6 +3254,12 @@
 				{"searchTerm": "Kagero - Kokumeikan Shinshou"}
 			]
 		},
+		{

```

Comment:
What would you like to achieve with this change? With only one title in the list, this won't do anything.